src/data/ACTIONS/root/SMN.ts

```diff
@@ -157,6 +157,15 @@ export const SMN = ensureActions({
 		cooldown: 20000,
 	},
 
+	RUBY_RUIN_III: {
```

Comment:
This will only give Ruby Ruin III a 2.8s GCD, not the full 3.0s it has in the job guide.  I didn't check the patch notes to be sure, but I assume this changed in patch 6.1, like it did for Ruby Rite, so you will need to add something to the 6.1 patch file (src\data\ACTIONS\layers\patch6.1.ts) for that.

Reply:
I checked the patch notes - it started with 2.8s cast, 2.5s recast in 6.0, then became 2.8 cast, 3.0s recast in 6.1. I'll update it :)